sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTraceExporter.cs

```diff
@@ -43,6 +42,10 @@ public override ExportResult Export(in Batch<Activity> batch)
                 var telemetryItems = TraceHelper.OtelToAzureMonitorTrace(batch, TraceResource, _instrumentationKey);
                 if (telemetryItems.Count > 0)
                 {
+                    if (TraceResource?.MetricTelemetry != null)
```

Comment:
Should we move this above `if (telemetryItems.Count > 0)`?

Reply:
If we change it, collected trace won't be sent to breeze if `TraceResource?.MetricTelemetry` is null.